spec/services/topic_status_updater_spec.rb

```diff
@@ -91,15 +91,12 @@
 
       before do
         # auto close after 3 days, topic was created a day ago
-        topic.update(
-          category:
-            Fabricate(
-              :category,
-              auto_close_hours: 72,
-              auto_close_based_on_last_post: based_on_last_post,
-            ),
-          created_at: 1.day.ago,
-        )
+        topic.update(category: Fabricate(:category), created_at: 1.day.ago)
+        topic.category.set_or_create_default_timer duration_minutes: 72 * 60,
+                                                   execute_at: Time.now,
+                                                   status_type: BaseTimer.types[:close],
+                                                   user: Discourse.system_user,
+                                                   based_on_last_post: based_on_last_post
```

Comment:
Instead of doing this, let's introduce a `category_default_topic_timer` fabricator instead.